tests/testMaskedImage.py

```diff
@@ -120,9 +120,10 @@ def tearDown(self):
 
     def testArrays(self):
         image, mask, variance = self.mimage.getArrays()
-        self.assert_((self.mimage.getImage().getArray() == image).all())
-        self.assert_((self.mimage.getMask().getArray() == mask).all())
-        self.assert_((self.mimage.getVariance().getArray() == variance).all())
+        self.assertListEqual(self.mimage.getImage().getArray().flatten().tolist(), image.flatten().tolist())
```

Comment:
Here are the docs I'm using:

    assertListEqual(list1, list2, msg=None)
    assertTupleEqual(tuple1, tuple2, msg=None)
        Tests that two lists or tuples are equal. If not, an error message is constructed that shows only the differences between the two. An error is also raised if either of the parameters are of the wrong type. These methods are used by default when comparing lists or tuples with assertEqual().

Based on this it is clear that `assertEqual` calls these specialized versions if appropriate, but less clear if that means `assertEqual` is usually preferred. I tend to prefer `assertEqual`, but I can see situations where the longer name might make it clearer what is going on.